### PR TITLE
Use bbcodebundle

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-front-demo/pull/101
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1042
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/170

### CODE REVIEW

File: BBcodeBundle/Definition/BBcodeDefinitionFactory.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        $this->className = $className;
+    }
+
+    /**
+     * Create a new definition
+     * 
+     * @param string  $tag
+     * @param string  $html
+     * boolean        $useOption
```
Comment (itkg-nanne) [inline BBcodeBundle/Definition/BBcodeDefinitionFactory.php]: no need @param?

**Comment by itkg-nanne** [comment]
:+1:

File: BBcodeBundle/Definition/BBcodeDefinitionFactory.php (hunk 1 above)
Comment (itkg-ngilain) [inline BBcodeBundle/Definition/BBcodeDefinitionFactory.php]: You're right, it's a failure, I fix that

**Comment by itkg-alavieille** [comment]
why remove all tests ?

**Comment by itkg-ngilain** [comment]
@itkg-alavieille : they are useless now, but I'll add new ones when all this part will be stabilized (soon so :) )

**Pull request merged**